ezissue/converter/manipulation.py

```diff
@@ -82,11 +82,20 @@ def format_acc_criteria(string):
     Formats the string adding the acceptance criteria header and adds a final
     new line.
     """
-    checkboxes = add_md_checkbox(string)
-    acc_criteria_title = "**Acceptance criteria:**\n---\n\n"
+    checkboxes = add_md_checkbox(string) if string != 'Não há critérios.\n' else string
```

Comment:
Shouldn't this logic be inside the `add_md_checkbox` function? It is simple, yes, but inside the function would be better organized